mirdata/jams_utils.py

```diff
@@ -88,10 +88,10 @@ def jams_converter(
 
         try:
             with open(audio_path, "rb") as fhandle:
-                duration = librosa.get_duration(filename=fhandle)
+                duration = librosa.get_duration(path=fhandle)
         # for local mp3s only
         except TypeError:
-            duration = librosa.get_duration(filename=audio_path)
+            duration = librosa.get_duration(path=audio_path)
```

Comment:
@genisplaja I'm not sure what's the motivation behind reading the audio using `with open()` to get the duration later rather than using librosa.load... Also, not sure why now it's not passing codecov and previously it was.

Reply:
How is that standing now? Should we still take a look?

Reply:
this is ok, now. I don't know what happened the other time I tried, but seems ok now. Actually it is a future deprecation warning from librosa, so good to incorporate that in this PR